utils/generator/templates/struct_function_template.java

```diff
@@ -0,0 +1,69 @@
+{% extends "base_template.java" %}
+
+{% block body %}
+public class {{class_name}} extends {{extends_class}} {
+    {%- if params is defined and ((kind is defined and kind != "response") or kind is not defined) %}
+    {%- for p in params %}
+    {%- if p.see is defined or p.deprecated is not none %}
+    /**
+     {%- if p.deprecated is not none %}
+     * @deprecated
+     {%- endif %}
+     {%- if p.see is defined %}
+     * @see {{p.see}}
+     {%- endif %}
+     */
+    {%- endif %}
+    {%- if p.deprecated is not none %}
+    @Deprecated
+    {%- endif %}
+    {%- if p.name is defined %}
+    private {% if p.modifier is defined %}{{p.modifier}} {% endif %}{{p.return_type}} {{p.name}}{% if p.value is defined %} = {{p.value}}{% endif %};
```

Comment:
I am a little confused about what that generates exactly.

Reply:
I didn't understand about what exactly line of code you are talking about, e.g. following xml:
```xml
<struct name="DisplayCapabilities" deprecated="true" since="6.0">
        <param name="displayType" type="DisplayType" mandatory="true" deprecated="true" since="5.0">
```
will be represent as following java code:
```java
@Deprecated
public class DisplayCapabilities extends RPCStruct {
    /**
     * @deprecated
     */
    @Deprecated
    public static final String KEY_DISPLAY_TYPE = "displayType";
```
If you're mean following: 
```jinja
{% if p.modifier is defined %}{{p.modifier}} {% endif %}
```
I guess it was added for some kind of manual customisation, and after we decide to do any manual customisation I believe it also useless.
Deleting it.

Reply:
I am referring to lines 20 & 21. It is adding a field with a `private` modifier. However, I am not sure in which case an RPC can have a field with a private modifier.

Reply:
```jinja
.name
```
was using for manual customisation.
Deleting it.